api/multicluster/v1alpha1/cluster.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+package multicluster.solo.io;
+
+import "gogoproto/gogo.proto";
+option (gogoproto.equal_all) = true;
+import "extproto/ext.proto";
+option (extproto.hash_all) = true;
+
+option go_package = "github.com/solo-io/skv2/pkg/api/multicluster.solo.io/v1alpha1/types";
+
+
+
+/*
+    Representation of a Kubernetes cluster that has been registered.
+*/
+message KubernetesClusterSpec {
+    // name of the secret which contains the kubeconfig with information to connect to the remote cluster.
+    string secret = 1;
+    // version of kubernetes
+    string version = 2;
+    // cloud provider, empty if unknown
+    string cloud = 3;
+
+    string region = 4;
+
+    string zone = 5;
```

Comment:
are any of these other fields used anywhere? how are they intended to be consumed?